### PR TITLE
Docker

### PR DESCRIPTION
**Pull request by SirMorfield**

### CODE REVIEW

File: src/config.php [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<?php
+$clientID = "581984985585436d517d906444b0c8c75120411a74d6d5891e02efd5d8bb1942";
+$clientSecret = "e962896f0cc16915625478a3400bac3fb6fdf55631666abfb789bfc2b2a188c5";
```
Comment (FreekBes) [inline src/config.php:3]: Don't push your client secret...

File: entrypoint.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#/bin/bash
+
+env php src/check.php &
+php -S 0.0.0.0:8080
```
Comment (FreekBes) [inline entrypoint.sh:4]: Would probably be better to have an nginx server to host the website on, and run the src/check.php file using the systemd service instead

**Review by FreekBes — CHANGES_REQUESTED** [review]
Please address the changes commented:
- remove the client secret and app id
- switch to nginx and systemd services

File: src/config.php (hunk 1 above)
Comment (SirMorfield) [reply src/config.php:3]: Hope that harm doesn't see it

**Pull request closed without merging**